vividus-plugin-web-app/src/main/resources/properties/profile/web/docker/profile.properties

```diff
@@ -2,3 +2,5 @@ selenium.grid.enabled=true
 selenium.grid.capabilities.platformName=linux
 selenium.grid.capabilities.browserName=${selenium.browser}
 selenium.grid.url=http://localhost:4444/wd/hub
+
+web.driver.CHROME.experimental-options={"prefs": {"profile": {"cookie_controls_mode": 2}}}
```

Comment:
it should be moved to documentation, create a new section, e.g. `How to configure Chrome browser settings`

Reply:
done